client/src/components/InputDetailCard.js

```diff
@@ -0,0 +1,136 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import GlobalStyles from './GlobalStyles';
+import MethodButton from './MethodButton';
+import CategoryButton from './CategoryButton';
+
+const Card = styled.div`
+  width: 343px;
+  height: 570px;
+  border-radius: 10px 10px 10px 10px;
+  border: 3px solid #b9b9b9;
+  color: white;
+  display: grid;
+  grid-template-rows: 10% 8% 8% 8% auto 10%;
+  grid-template-columns: 15% 15% auto 15% 7%;
+  align-items: center;
+`;
+
+const Heading = styled.input`
+  width: 240px;
+  height: 38px;
+  border: 1px solid #011e62;
+  font-size: 20px;
+  padding-left: 8px;
+  color: #011e62;
+  margin-left: 15px;
+`;
+
+const SubHeading = styled.h2`
+  font-size: 1.125rem;
```

Comment:
you are using px above in the Heading and here rem.. and also a bit weird number. rather use 1.1 or something maybe?

Reply:
will fix that in the next refactoring :)